services/storehost/inConn.go

```diff
@@ -303,7 +304,7 @@ func (t *inConn) writeMessagesPumpAppendOnly(msgC <-chan *inMessage, ackC chan<-
 			}
 
 			// if seqNum is not in expected (continuously increasing) order, log error as FYI
-			if x.getLastSeqNum()+1 != msgSeqNum && x.getLastSeqNum() != -1 {
+			if x.getLastSeqNum()+1 != msgSeqNum && x.getLastSeqNum() != math.MaxInt64 {
```

Comment:
You might be better off defining a const for invalid first/last {seq,addr,ts} values. const helps if (1) ever you change this value to something else other than math.MaxInt64 (2) Finding out all references where invalid_ values are accessed in code.